includes/tracks/class-wc-tracks-client.php

```diff
@@ -55,29 +55,22 @@ public static function record_event( $event ) {
 	 * Synchronously request the pixel.
 	 *
 	 * @param string $pixel pixel url and query string.
-	 * @return bool|WP_Error         True on success, WP_Error on failure.
+	 * @return bool Always returns true.
 	 */
 	public static function record_pixel( $pixel ) {
 		// Add the Request Timestamp and URL terminator just before the HTTP request.
 		$pixel .= '&_rt=' . self::build_timestamp() . '&_=_';
 
-		$response = wp_safe_remote_get(
+		wp_safe_remote_get(
 			$pixel,
 			array(
-				'blocking'    => true, // The default, but being explicit here.
+				'blocking'    => false,
 				'redirection' => 2,
 				'httpversion' => '1.1',
+				'timeout'     => 1,
 			)
 		);
 
-		if ( is_wp_error( $response ) ) {
-			return $response;
-		}
-
-		if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
-			return new WP_Error( 'request_failed', 'Tracks pixel request failed', $code );
```

Comment:
Just a question, why not keep tracking it?
We could also include this in the log system.

Reply:
We aren't using the return of `record_event()` anywhere and I was trying to get this function to return ASAP. Ideally firing off a non blocking request without processing the response should be faster, but I'm not 100% sure that's the case in WP's HTTP implementation.